libs/geography/src/models/address.ts

```diff
@@ -1,13 +1,40 @@
 /**
- * A basic model of an address
+ * A basic model of an address.
+ * Either a region or region_id must be supplied.
+ * Addresses should either satisfy DaffUnknownRegionAddress or DaffKnownRegionAddress.
  */
 export interface DaffAddress {
 	street: string;
 	street2?: string;
   city: string;
-  region: string;
-  region_id?: string | number;
-  country?: string;
-  country_id?: string | number;
   postcode: string;
-}
\ No newline at end of file
+  country_id: string | number;
+  /**
+   * A field for a region_id supplied by the platform.
+   * Needed for countries that have a list of known regions.
+   * Required if region is not set.
+   */
+  region_id?: string | number;
+  /**
+   * A field for user-supplied regions.
+   * Needed for countries that don't have any regions supplied by the platform.
```

Comment:
I think this comment could be a little more precise. "Needed for countries that have regions not supplied by the platform."

Reply:
ah nice, that is better. Changed